crates/forge_analyzer/src/utils.rs

```diff
@@ -0,0 +1,15 @@
+use crate::definitions::CalleeRef;
+use swc_core::ecma::ast::{CallExpr, Callee, Expr, MemberProp};
+
+pub fn call_func_with_name(n: &CalleeRef, name: &str) -> bool {
```

Comment:
```suggestion
pub fn calls_method(n: CalleeRef<'_>, name: &str) -> bool {
```
`CalleeRef` is essentially already a reference, so it's already `Copy`. So passing in a reference to `CalleeRef` is just double indirection.